packages/remix-cloudflare-pages/worker.ts

```diff
@@ -53,16 +53,19 @@ export function createPagesFunctionHandler<Env = any>({
     // https://github.com/cloudflare/wrangler2/issues/117
     context.request.headers.delete("if-none-match");
 
-    try {
-      response = await context.env.ASSETS.fetch(
-        context.request.url,
-        context.request.clone()
-      );
-      response =
-        response && response.status >= 200 && response.status < 400
-          ? new Response(response.body, response)
-          : undefined;
-    } catch {}
+    let method = context.request.method;
+    if (method !== "POST" && method !== "PUT") {
```

Comment:
Probably would be better to positively match `"GET"` and `"HEAD"` rather than exclude.

Reply:
Yeah, I wasn't possible that there wouldn't be some other verb that might be used legitimately.. like OPTIONS, maybe? Probably not. I can change it.

Reply:
I changed it to check for GET/HEAD/OPTIONS.